tex/chapters/07-ApplyingModel.tex

```diff
@@ -0,0 +1,197 @@
+%!TEX root = ../main.tex
+
```

Comment:
An SVM model is defined by many vectors and in our case it uses 596 vectors, each of which is of length 12.

It is possible to get the parameters that sk-learn is using which includes things like the underlying kernel